profiles/kentik_snmp/apc/apc_ats.yml

```diff
@@ -0,0 +1,346 @@
+      - OID: 1.3.6.1.4.1.318.1.1.8.5.4.3.1.15
+        name: atsOutputMinPower
+      - OID: 1.3.6.1.4.1.318.1.1.8.5.4.3.1.16
+        name: atsOutputPercentPower
+      - OID: 1.3.6.1.4.1.318.1.1.8.5.4.3.1.17
+        name: atsOutputMaxPercentPower
+      - OID: 1.3.6.1.4.1.318.1.1.8.5.4.3.1.18
+        name: atsOutputMinPercentPower
+      - OID: 1.3.6.1.4.1.318.1.1.8.5.4.3.1.19
+        name: atsOutputPhaseState
+        enum:
+          normal: 1
+          lowload: 2
+          nearoverload: 3
+          overload: 4
+    metric_tags:
+      - tag: output_phase_index
+        column:
+          OID: 1.3.6.1.4.1.318.1.1.8.5.4.3.1.1
+          name: atsOutputPhaseTableIndex
+      - tag: output_phase_number
+        column:
+          OID: 1.3.6.1.4.1.318.1.1.8.5.4.3.1.2
+          name: atsOutputPhaseIndex
+  - MIB: PowerNet-MIB_ATS
+    table:
+      OID: 1.3.6.1.4.1.318.1.1.8.5.4.5
+      name: atsOutputBankTable
+    symbols:
+      - OID: 1.3.6.1.4.1.318.1.1.8.5.4.5.1.2
+        name: atsOutputPhase
+        enum:
+          phase1: 1
+          phase2: 2
+          phase3: 3
+      - OID: 1.3.6.1.4.1.318.1.1.8.5.4.5.1.3
+        name: atsOutputBank
+        enum:
+          total: 1
+          bank1: 2
+          bank2: 3
+      - OID: 1.3.6.1.4.1.318.1.1.8.5.4.5.1.4
+        name: atsOutputBankCurrent
+      - OID: 1.3.6.1.4.1.318.1.1.8.5.4.5.1.5
+        name: atsOutputBankState
+        enum:
+          normal: 1
+          lowload: 2
+          nearoverload: 3
+          overload: 4
+      - OID: 1.3.6.1.4.1.318.1.1.8.5.4.5.1.6
+        name: atsOutputBankOutputVoltage
+      - OID: 1.3.6.1.4.1.318.1.1.8.5.4.5.1.7
+        name: atsOutputBankMaxCurrent
+      - OID: 1.3.6.1.4.1.318.1.1.8.5.4.5.1.8
+        name: atsOutputBankMinCurrent
+      - OID: 1.3.6.1.4.1.318.1.1.8.5.4.5.1.9
+        name: atsOutputBankLoad
+      - OID: 1.3.6.1.4.1.318.1.1.8.5.4.5.1.10
+        name: atsOutputBankMaxLoad
+      - OID: 1.3.6.1.4.1.318.1.1.8.5.4.5.1.11
+        name: atsOutputBankMinLoad
+      - OID: 1.3.6.1.4.1.318.1.1.8.5.4.5.1.12
+        name: atsOutputBankPercentLoad
+      - OID: 1.3.6.1.4.1.318.1.1.8.5.4.5.1.13
+        name: atsOutputBankMaxPercentLoad
+      - OID: 1.3.6.1.4.1.318.1.1.8.5.4.5.1.14
+        name: atsOutputBankMinPercentLoad
+      - OID: 1.3.6.1.4.1.318.1.1.8.5.4.5.1.15
+        name: atsOutputBankPower
+      - OID: 1.3.6.1.4.1.318.1.1.8.5.4.5.1.16
+        name: atsOutputBankMaxPower
+      - OID: 1.3.6.1.4.1.318.1.1.8.5.4.5.1.17
+        name: atsOutputBankMinPower
+      - OID: 1.3.6.1.4.1.318.1.1.8.5.4.5.1.18
+        name: atsOutputBankPercentPower
+      - OID: 1.3.6.1.4.1.318.1.1.8.5.4.5.1.19
+        name: atsOutputBankMaxPercentPower
+      - OID: 1.3.6.1.4.1.318.1.1.8.5.4.5.1.20
+        name: atsOutputBankMinPercentPower
+    metric_tags:
+      - tag: output_bank_table_index
+        column:
+          OID: 1.3.6.1.4.1.318.1.1.8.5.4.5.1.1
+          name: atsOutputBankTableIndex
```

Comment:
this table isn't seen in the results of the SNMP walk, are we confident that it's supported in other ATS models?

Reply:
This is one I'm not 100% confident on. It's in the MIB, and my ATS shows `0` for `atsOutputBankTableSize (1.3.6.1.4.1.318.1.1.8.5.4.4)`, which is appropriate, since I only have a single-output ATS. Unfortunately, I don't have a multi-bank ATS to test this on.

Reply:
I went ahead and updated this table to match the other table suggestions below.